src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -150,7 +139,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <PrivateSdkAssemblies Include="$(IlcPath)\sdk\*.dll" />
     </ItemGroup>
     <ItemGroup>
-      <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" />
+      <!-- Exclude System.IO.Compression.Native.dll for now https://github.com/dotnet/corert/issues/5496 -->
+      <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" Exclude="$(IlcPath)\framework\System.IO.Compression.Native.dll" />
```

Comment:
```suggestion
      <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" Exclude="$(IlcPath)\framework\*.Native.dll" />
```
Future proofing

Reply:
This was a copypaste from a different spot in this file. I fixed both.